Examples/UtiliKit-iOSExample/ViewController/ActiveLabelViewController.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ActiveLabelViewController.swift
+//  UtiliKit-iOS
+//
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import UIKit
+
+class ActiveLabelViewController: UIViewController {
+    @IBOutlet private var activeLabels: [ActiveLabel]!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        activeLabels.forEach({ $0.text = nil })
```

Comment:
I also thought about having a bool that would allow you to enable/disable the automatic animation when the label is set to nil. My worry was just that it might over complicate the use of this. I will go ahead and explore that though since not having that ability does limit the enduser when using this class.

Reply:
Yeah, I can see how it would make things more complicated. I don't necessarily think we need to solve it right now though, since I just made up the scenario in my head. 😄